### PR TITLE
New package: wayvnc-0.2.0

### PR DESCRIPTION
**Pull request by SevereOverfl0w**

### CODE REVIEW

**Comment by ericonr** [comment]
Duplicate of #23884

**Comment by SevereOverfl0w** [comment]
I didn't spot that, I started working on this a couple weekends ago (I prodded for the 0.1.0 aml release).

This package has the -devel in place.  I didn't know musl was broken, but I've been able to copy the fix from #23884, and I'm just testing that out now.

My PR doesn't have split commits, but I can easily add that if that's how it should be done.

**Comment by ericonr** [comment]
Yes, it should definitely have split commits.

**Comment by SevereOverfl0w** [comment]
That's been fixed now.  Separate commits.

File: srcpkgs/wayvnc/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'wayvnc'
+pkgname=wayvnc
+version=0.2.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
```
Comment (fosslinux) [inline srcpkgs/wayvnc/template:6]: You need wayland-devel here too for wayland-scanner IIRC.

**Comment by SevereOverfl0w** [comment]
wayland-devel added :)

**Comment by ahesford** [comment]
The author of #23884 is adding `-devel` subpackages, so we'll finish this in the earlier PR.

**Pull request closed without merging**